app/admin/dashboard.rb

```diff
@@ -27,6 +27,17 @@
           button_to "Go into site-wide read-only mode", toggle_read_only_mode_admin_site_settings_path
         end
       end
+      para do
+        form action: update_maximum_concurrent_scrapers_admin_site_settings_path, method: :post do
+          div do
+            label 'Maximum concurrent scrapers'
```

Comment:
I think it's non-trivial to give a recommended value with the things configured how they are currently. For example, even if you know the system memory you don't necessarily know the memory used by all the non-docker stuff on the machine. So, I think the best thing we can really do for now is to add a little helper text to just remind the admin of the amount of memory that each scraper can potentially use. I've added another commit to that effect